ui/propertyeditor/propertyeditorfactory.cpp

```diff
@@ -45,6 +45,7 @@ PropertyEditorFactory::PropertyEditorFactory()
     initBuiltInTypes();
 
     addEditor(QVariant::Color, new QStandardItemEditorCreator<PropertyColorEditor>());
+    addEditor(QVariant::ByteArray, new QStandardItemEditorCreator<PropertyTextEditor>());
```

Comment:
I did check nothing but is not the editor would change the QByteArray type to a QString here ?